packages/dds/tree/api-report/tree.api.md

```diff
@@ -41,17 +41,14 @@ export interface Adapters {
     readonly tree?: readonly TreeAdapter[];
 }
 
-// @internal
-export type AllowedTypes = readonly [Any] | readonly LazyItem<FlexTreeNodeSchema>[];
-
-// @beta
-type AllowedTypes_2 = readonly LazyItem<TreeNodeSchema>[];
+// @public
+export type AllowedTypes = readonly LazyItem<TreeNodeSchema>[];
```

Comment:
Fixed missing export for the new 'AllowedTypes'.

When you forget to export, you get a non-exported type declaration as shown in the red.

Non-exported declarations break when consuming the types from trimmed rollups.

We typically catch this in the 'fluid-framework' package, which consumes all types by re-exporting:
```ts
export * from "@fluidframework/tree";
```